hooks/usePoll.ts

```
@@ -10,47 +16,60 @@ interface UsePollParams {
-          sortOrder: filters.sortOrder || "asc",
-          isActive: String(filters.isActive ?? undefined),
-          userVoted: String(filters.userVoted ?? undefined),
-          userCreated: String(filters.userCreated ?? undefined),
-        });
+export const getPolls = (
+  filters: IUsePollParams = {}
+): UseQueryResult<{
+  polls: IPoll[];
+  total: number;
+}> => {
+  return useQuery({
```

ESLint

[error] React Hook "useQuery" is called in function "getPolls" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)